### PR TITLE
Fix import/export of episode bookmarks

### PR DESCRIPTION
**Pull request by phate89**
You need playerstate to resume disc episode bookmark.

### CODE REVIEW

**Comment by Montellese** [comment]
Did `CVideoInfoTag::m_fEpBookmark` ever work for anything if you need the playerstate as well?

**Comment by phate89** [comment]
Playerstate is needed for dvd  (and maybe bluray) only. Time only is safe for the rest. There's the same problem with exporting resume state and upnp iso resume

**Comment by Montellese** [comment]
So in a lot of cases the old approach is still valid? If so we might want to keep backwards compatibility by also parsing the old `epbookmark` XML tag from NFOs. So first check if there's an `episodebookmark` tag available and if not, try to at least read the `timeInSeconds` value from the old `epbookmark` tag.

**Comment by phate89** [comment]
Unless I did something wrong is what I'm doing. I check if episode bookmark exist. If it doesn't I import epbookmark

**Comment by Montellese** [comment]
Ah sorry I only saw that the line responsible for parsing of `epbookmark` has been removed and missed the `else if` block. All good then.

File: xbmc/video/VideoInfoTag.cpp [hunk 1]
```diff
@@ -726,7 +743,24 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
...
+    const TiXmlElement *playerstate = epbookmark->FirstChildElement("playerstate");
+    if (playerstate)
+    {
+      const TiXmlElement *value = playerstate->FirstChildElement();
+      if (value)
+        m_EpBookmark.playerState << *value;
+    }
+  }
+  else
+  {
```
Comment (Montellese) [inline xbmc/video/VideoInfoTag.cpp]: We don't use curly braces on single line blocks.

Comment (phate89) [inline xbmc/video/VideoInfoTag.cpp]: It's ok to have if with curly braces and else without it? I thought it wasn't ok.. I'll fix it

**Comment by phate89** [comment]
jenkins build this please

**Comment by Razzeee** [comment]
Builds were fine

**Pull request merged**